src/document/json/object-map-html.xsl

```diff
@@ -216,7 +216,7 @@ details:not([open]) .show-closed { display: inline }
       <xsl:if test="empty(@_tree-json-id)">
          <xsl:message expand-text="true">not seeing json tree id for { name(.) }</xsl:message>
       </xsl:if>
-      <a class="OM-name" href="{ $reference-link }#{  @_tree-json-id }">
+      <a class="OM-name{ if (@deprecated) then ' OM-name-deprecated' else ''}" href="{ $reference-link }#{  @_tree-json-id }">
```

Comment:
A similar optimization, assuming you call it that: `{ @deprecated/' OM-name-deprecated' }`

(If you think it's too obscure the idiom can also be commented.)